packages/pipeline-editor/src/index.tsx

```diff
@@ -74,590 +42,6 @@ const commandIDs = {
   newDocManager: 'docmanager:new-untitled'
 };
```

Comment:
We should export this from `Pipeline.tsx` and import it here rather than redeclare it (that's how it's done in lab)